packages/aragon-wrapper/src/index.js

```diff
@@ -969,13 +977,15 @@ export default class Aragon {
         if (await this.canForward(forwarder, sender, script)) {
           // The previous forwarder can forward a transaction for this forwarder,
           // and this forwarder can forward for our address, so we have found a path
-          return [await createForwarderTransaction(forwarder, script), ...path]
+          const transaction = createForwarderTransaction(forwarder, script)
+          // `applyTransactionGas` is only done for the transaction that will be executed
+          return [await this.applyTransactionGas(transaction, true), ...path]
```

Comment:
Note that the forwarding transaction here and above can still fail, but given the complexity of making sure this robust, I'm fine with leaving it like this for now. I've added TODO comments to remind us.